backend/app.js

```diff
@@ -1,24 +1,29 @@
 const express = require('express');
```

Comment:
Rule 1 of working with package.json should be checking if the node_modules are being ignored